tests/python/release_checklist/check_graph_view.py

```diff
@@ -22,6 +27,27 @@ def log_readme() -> None:
     rr.log("readme", rr.TextDocument(README, media_type=rr.MediaType.MARKDOWN), static=True)
 
 
+def log_weird_graph() -> None:
```

Comment:
Can we have a better name for this? "same-node edges", "self-connected nodes", sth else?

Reply:
Also, best split that checklist in two. The instruction quickly become indigestible wall of text and it takes brain power to find the right views to look at.